### ISSUE TITLE
missing the formula 8.

### ISSUE THREAD

**Issue by qinyanjun04** [NLPWM-WHU/LSTPM#6]
The formula 8 is missing in the paper?

**Comment by John-Almardeny** [comment]
Yeah, I saw it. The whole paper and results are just a pile of crab lol
It's almost always the case for any Chinese work!
